x-pack/plugins/security_solution/public/common/lib/kuery/index.ts

```diff
@@ -231,6 +231,12 @@ export const convertToBuildEsQuery = ({
   }
 };
 
+export interface CombinedQuery {
+  filterQuery: string | undefined;
+  kqlError: Error | undefined;
+  baseKqlQuery: Query;
```

Comment:
Added the "raw" `baseKqlQuery` to the return type since it's used for the timeline data service.